localstack/config.py

```diff
@@ -655,6 +655,8 @@ def populate_edge_configuration(
     GATEWAY_LISTEN,
 ) = populate_edge_configuration(os.environ)
 
+GATEWAY_WORKER_THREAD_COUNT = int(os.environ.get("GATEWAY_WORKER_THREAD_COUNT") or 1000)
```

Comment:
naming seems fine to me :+1: could argue `GATEWAY_WORKER_COUNT` to make it more generic but threads is more accurate :+1: